tests/plugins/a11yhelp/keystrokes.js

```diff
@@ -0,0 +1,230 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: a11yhelp,dialog*/
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
```

Comment:
You're actually not using this line. This line creates a default editor, but you're creating a custom instance each time in `_createEditor` method. So it can be removed.